modules/rosa-hcp-sts-creating-a-cluster-ext-auth-provider-cli.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with the enabled option for external authentication providers, you must create a provider using the ROSA CLI.
+
+[NOTE]
+====
+Similar to the `rosa create|delete|list idp[s]` command in the ROSA CLI, you cannot edit an existing identity provider that you created using `rosa create external-auth-provider`. Instead, you must delete the external authentication provider and create a new one. See the additional resources for more information about the `idp` subcommand of the ROSA CLI.
+====
+
+This configuration uses Microsoft Entra ID to authenticate your credentials. The Entra ID values are:
+
+[cols="1,1", options="header"]
+|===
+
+|CLI Flag
+|Expected value
+
+|`--cluster` 
+|This string is the name or the ID of your cluster
+
+|`--name`
+|This string is the name you set for your external authentication provider.
+
+|`--console-client-secret`
+|This string is the client secret that is used to associate your account with the application. If you do not include the client secret, this command uses a public OIDC OAuthClient.
+
+|`--issuer-audiences` 
+|This common-separated list of strings is your client ID. In Microsoft Entra ID, this is called the audience ID.
```

Comment:
This is mapped with this attribute:
https://github.com/openshift/api/blob/master/config/v1/types_authentication.go#L240